etc/ProdOfflineConfiguration.py

```diff
@@ -682,6 +700,23 @@
                physics_skims=["TopMuEG", "LogError", "LogErrorMonitor"],
                scenario=ppScenario)
 
+DATASETS = ["Muon"]
+
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               write_reco=False,
+               write_dqm=True,
+               tape_node="T1_US_FNAL_MSS",  # SingleMon was in "T1_US_FNAL_MSS" , DoubleMuon was in "T1_DE_KIT_MSS"
+               disk_node="T1_US_FNAL_Disk", # SingleMon was in "T1_US_FNAL_Disk", DoubleMuon was in "T1_DE_KIT_Disk"
```

Comment:
same comment here: the tape/disk_nodes are take from Single Muon, should we update them @drkovalskyi?

Reply:
As before. I think it's ok to keep them as this until next week's rebalance.